packages/hash/frontend/src/blocks/page/PageTitle.tsx

```diff
@@ -70,16 +94,34 @@ export const PageTitle: FunctionComponent<PageTitleProps> = ({
 
   // TODO: Assign appropriate a11y attributes
   return (
-    <input
-      placeholder="A title for the page"
-      disabled={updateEntityLoading}
-      onChange={handleInputChange}
+    <Typography
```

Comment:
I think it would probably be cleaner to just use a MUI TextField and style it accordingly. It prevents bold and italicized text by default and we could avoid using a ref as well.